virttest/libvirt_xml/network_xml.py

```diff
@@ -484,10 +486,12 @@ def del_defined(self):
 
     def get_active(self):
         """Accessor method for 'active' property (True/False)"""
-        self.__check_undefined__("Cannot determine activation for undefined "
-                                 "network")
         state_dict = self.virsh.net_state_dict(virsh_instance=self.virsh)
-        return state_dict[self.name]['active']
+        try:
+            active_state = state_dict[self.name]['active']
```

Comment:
return state_dict[self.name]['active'] directly, and line 494 can be leave out

Reply:
It is for catching error